test/Dialect/FIRRTL/check-comb-cycles.mlir

```diff
@@ -1,10 +1,12 @@
-// RUN: circt-opt --pass-pipeline='builtin.module(firrtl.circuit(firrtl-check-comb-loops))' --split-input-file --verify-diagnostics %s | FileCheck %s
+// RUN: circt-opt --pass-pipeline='builtin.module(firrtl.circuit(firrtl-check-comb-loops))' --split-input-file  --verify-diagnostics %s | FileCheck %s
 
 // Loop-free circuit
 // CHECK: firrtl.circuit "hasnoloops"
 firrtl.circuit "hasnoloops"   {
   firrtl.module @thru(in %in1: !firrtl.uint<1>, in %in2: !firrtl.uint<1>, out %out1: !firrtl.uint<1>, out %out2: !firrtl.uint<1>) {
-    firrtl.connect %out1, %in1 : !firrtl.uint<1>, !firrtl.uint<1>
+    %a = firrtl.wire : !firrtl.uint<1>
+    firrtl.connect %out1, %a : !firrtl.uint<1>, !firrtl.uint<1>
```

Comment:
Just curious what this change is for?

Reply:
Don't recall sorry, :P, might have been to increase code coverage.